pyaerocom/griddeddata.py

```diff
@@ -676,6 +676,10 @@ def register_var_glob(self, delete_existing=True):
             f'to add this variable to pyaerocom/data/variables.ini.')
         return vardef
 
+
```

Comment:
not sure if I want this warning to disappear. Might make error search (why data was not read more complicated)

Reply:
Then we need to ignore it on the test, or in the pytest configuration